strawberry/subscriptions/__init__.py

```diff
@@ -1,2 +1,8 @@
+from typing_extensions import Literal
+
 GRAPHQL_TRANSPORT_WS_PROTOCOL = "graphql-transport-ws"
 GRAPHQL_WS_PROTOCOL = "graphql-ws"
+
+SubscriptionProtocolType = Literal[  #  type: ignore
+    GRAPHQL_TRANSPORT_WS_PROTOCOL, GRAPHQL_WS_PROTOCOL  #  type: ignore
+]
```

Comment:
I'm curious about why we need the type ignores here. Doesn't CI not pass without them?

Reply:
Yes it's a bit weird, mypy seems to complain about the use of literal in literal, but it's considered as valid after [this pep](https://peps.python.org/pep-0586/#legal-parameters-for-literal-at-type-check-time)